elifetools/tests/fixtures/test_authors_json/content_02.xml

```diff
@@ -0,0 +1 @@
+<root xmlns:xlink="http://www.w3.org/1999/xlink"><front><article-meta><contrib-group><contrib contrib-type="author"><collab>ICGC Breast Cancer Group</collab></contrib></contrib-group></article-meta></front></root>
```

Comment:
I'm not sure if it changes anything about the result, but if possible I would try to store formatted XML content so that it's easy to view in an editor?

Reply:
Almost all of the XML we parse is collapsed on one line, although that doesn't mean all the specific test examples must also be that way. 

The places where extra new lines, spaces or tabs can make a difference is where inline tags in a paragraph are wrapped to the following line (using some XML pretty output) and as a result we will get extra whitespace in the results. For one example is to look at the eLife 00666 kitchen sink, and it has large strings of spaces where sentences span multiple lines in the XML file.

All that said, I'm happy to manually prettify the XML fixtures a certain amount, as long as it doesn't affect the output by adding extra spaces.